biome.jsonc

```
@@ -15,6 +15,7 @@
 			"**/dist/*",
 			"**/lib/*",
 			"**/*.done.build.log",
+			"**/.next/**/*", // NextJS
```

```suggestion
			"**/.next/**", // NextJS
```

That is equivalent, no?

---

note that globs are interpreted based on the working directory where biome is invoked, so you have to be sure the globs work in both places. In this case, it should work. To test, run biome in the root _and_ in the project folder and make sure there are no differences.

---

Yeah, in fact I'm keeping just `"**/.next"`. That matches other entries below like `"**/snapshots/content"`, and I just verified it works from the root and from the package folder.